app/assets/javascripts/admin/addon/components/admin-config-areas/components.gjs

```diff
@@ -47,14 +48,20 @@ const STATUS_FILTER_OPTIONS = [
 
 export default class AdminConfigAreasComponents extends Component {
   @service modal;
+
   @service router;
+
```

Comment:
Are those empty lines intentional? Usually, we group all services together similar to tracked properties

Reply:
Nope, not intentional, they've must been added by eslint when it rearranged the lines.